packages/core-commands/src/hooks.js

```diff
@@ -6,7 +6,7 @@ import { useSelect } from '@wordpress/data';
 
 export function useIsTemplatesAccessible() {
 	return useSelect(
-		( select ) => select( coreStore ).canUser( 'read', 'templates' ),
+		( select ) => select( coreStore ).canUser( 'create', 'templates' ),
```

Comment:
The name of the hook is not congruent with what it does. I understand that it's an internal hook so it's not that important but maybe the hook should be renamed?

Reply:
Indeed. For example, is `useCanCreateTemplates` appropriate?

Reply:
Works for me or even remove the hook entirely. Not sure how useful is it to abstract it.

Reply:
> remove the hook entirely

That makes sense. I would like to merge this PR after removing this hook and switching to an ad hoc approach.